backends/bmv2/run-bmv2-test.py

```diff
@@ -172,6 +178,8 @@ def run_model(options, tmpdir, jsonfile):
         return SUCCESS
 
     bmv2 = RunBMV2(tmpdir, options, jsonfile)
+    bmv2.cmd_line_args = options.switchOptions
```

Comment:
`options` is already passed in to RunBMV2 as an argument, there is no need for these two assignments outside the RunBMV2 function. Do the assignment inside RunBMV2?

Reply:
Will do.